### ISSUE TITLE
Vigilant spam on start - Detected GetModByName() return error value

### ISSUE THREAD

**Issue by alandtse** [Nightfallstorm/SkyrimVRESL#45]
Offending function below:

```
Function CheckSafe()
	int im = Game.GetModByName("Vigilant.esm")
	;Install SKSE ?
	if im == 0
		Debug.MessageBox("Vigilant:You need to install SKSE")

		return
	elseif im < 255
		String PlugInName = Game.GetModName(im) 
		if PlugInName != "Vigilant.esm"
			Debug.MessageBox("Vigilant: Detected GetModByName() return error value. There is an anomaly in Mod Manager indexing, ZMerge relinking, etc." + "\n" + "Index Number:" + im + "\n" + "Mod Name: " + PlugInName )
			return
		endif
	endif
```

**Comment by Nightfallstorm** [comment]
SE version:
```c++
	UInt32 GetModByName(StaticFunctionTag*, BSFixedString name)
	{
		DataHandler* pDataHandler = DataHandler::GetSingleton();
		const ModInfo * modInfo = pDataHandler->LookupModByName(name.data);
		if (!modInfo || !modInfo->IsActive())
			return 0xFF;

		if (modInfo->IsLight())
			return modInfo->lightIndex + LIGHT_MOD_OFFSET;

		return modInfo->modIndex; // modIndex of mod itself, not index in list of `files`
	}
```

VR version:
```c++
	UInt32 GetModByName(StaticFunctionTag*, BSFixedString name)
	{
		DataHandler* pDataHandler = DataHandler::GetSingleton();
		return pDataHandler->GetModIndex(name.data); // mod index in list of `files`, not the same as mod's actual index
	}
```

**Comment by Nightfallstorm** [comment]
fixed by #46
